<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeSettings</class>
 <widget class="QWidget" name="PracticeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PracticeSettings</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Име на практиката:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>РЗИ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Булстат:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Свободна професия</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ЕТ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ЕООД</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ООД</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ЕАД</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>АД</string>
    </property>
   </item>
  </widget>
  <widget class="LineEdit" name="practiceNameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="LineEdit" name="rziEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="LineEdit" name="bulstatEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>13</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Седалище:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес по дейност:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="activityAddressEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="nzokGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Договор с НЗОК</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="LineEdit" name="contractEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер на договора:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата на договора</string>
    </property>
   </widget>
   <widget class="DateEdit" name="contractDateEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>9999</year>
      <month>12</month>
      <day>30</day>
     </datetime>
    </property>
    <property name="minimumDate">
     <date>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="date">
     <date>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Съкратено име на практиката според договора:</string>
    </property>
   </widget>
   <widget class="LineEdit" name="practiceNameNZOK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="vatGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Регистрация по ДДС</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ДДС номер:</string>
    </property>
   </widget>
   <widget class="LineEdit" name="vatEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Форма:</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color : red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Парола:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>View/uiComponents/qt_derived/LineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>View/uiComponents/qt_derived/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>practiceNameEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>rziEdit</tabstop>
  <tabstop>bulstatEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>activityAddressEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>vatGroup</tabstop>
  <tabstop>vatEdit</tabstop>
  <tabstop>nzokGroup</tabstop>
  <tabstop>contractEdit</tabstop>
  <tabstop>contractDateEdit</tabstop>
  <tabstop>practiceNameNZOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
